static/js/services/update-user.js

```diff
@@ -1,145 +1,123 @@
-var _ = require('underscore');
-
 (function () {
 
   'use strict';
 
   var inboxServices = angular.module('inboxServices');
 
-  var getUserUrl = function(id) {
-    return '/_users/' + encodeURIComponent(id);
-  };
-
-  var removeCacheEntry = function($cacheFactory, id) {
```

Comment:
Do we never get the user information any more? What about admins?

Reply:
I couldn't work out how this cache was useful. I presume it's automatically used by `$http`, though it's not clear from documentation. We also only use it for this users. It also only applies to this one instance, and so if you leave your session open for a long time, or more than one user is modifying things, it presumably becomes invalid.

Maybe we need to talk about it, I certainly don't mean to remove useful functionality!